build-tools/xaprepare/xaprepare/ToolRunners/SevenZipRunner.cs

```diff
@@ -50,26 +50,28 @@ public async Task<bool> Extract (string archivePath, string outputDirectory, Lis
 
 		public async Task<bool> VerifyArchive (string archivePath)
 		{
-			if (String.IsNullOrEmpty (archivePath))
-				throw new ArgumentException ("must not be null or empty", nameof (archivePath));
-
-			ProcessRunner runner = CreateProcessRunner ("t");
-			AddStandardArguments (runner);
-			runner.AddQuotedArgument (archivePath);
-
-			try {
-				Log.StatusLine ($"Archive path: {archivePath}");
-				return await RunTool (
-					() => {
-						using (TextWriter outputSink = SetupOutputSink (runner, $"7zip-verify-archive.{Path.GetFileName (archivePath)}", "verifying archive integrity")) {
-							StartTwiddler ();
-							return runner.Run ();
-						}
-					}
-				);
-			} finally {
-				StopTwiddler ();
-			}
+			return true;
+
+			// if (String.IsNullOrEmpty (archivePath))
```

Comment:
Should we be commenting this out, or should we instead use `#if false` around the code?  Using the `#if` would result in less "noise" in the PR, but might be undesirable otherwise...